integration/mochaTests/README.md

```diff
@@ -0,0 +1,131 @@
+```
+
+SauceLabs will open a browser and load a test url. You need to have a serverless deployment of Chime JS SDK demo. You can set the demo url as an environment variable with the following command:
+```bash
+export TEST_URL=<JS SDK Demo URL>
+```
+
+Following command can be used to run a browser compatibility tests with default settings on SauceLabs:
+
+```bash
+npm run test -- --test-name AudioTest --host saucelabs --test-type browser-compatibility
+```
+
+There are scenarios where a test might not be compatible with one of the browsers or OS. In that case, user can provide a custom config with updated clients array. `sample_test.config.json` is a sample test config already provided. 
+Following command can be used to run a browser compatibility test with custom config:
+
+```bash
+npm run test -- --test-name AudioTest --host saucelabs --test-type browser-compatibility --config browserCompatibilityTest/desktop/sample_test.config.json
+```
+
+### Running Browser Compatibility Tests on DeviceFarm
+To run the tests on DeviceFarm, you will need an AWS account. Once you have an AWS account, you will need to setup two environment variables that will allow the AWS SDK to authenticate to your AWS account.
+```bash
+export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
+export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+```
+
+You will need to create a DeviceFarm project in the AWS account that you are planning to use. There are two types of devicefarm projects that can be created: `Mobile Device Testing` and `Desktop Browser Testing`. For this section, we will focus mainly on desktop browser testing.
+After the project is created you will need to setup a `PROJECT_ARN` as an environment variable. Project arn is used by the device farm API to identify the project to create test sessions inside.
+```bash
+export PROJECT_ARN=<YOUR_PROJECT_ARN>
+```
+
+Following command can be used to run a browser compatibility test with default settings on DeviceFarm:
+```bash
+npm run test -- --test-name AudioTest --host devicefarm --test-type browser-compatibility
+```
+
+Just like with SauceLabs, DeviceFarm can run browser compatibility tests with a custom config:
+```bash
+npm run test -- --test-name AudioTest --host devicefarm --test-type browser-compatibility --config browserCompatibilityTest/desktop/sample_test.config.json
+```
+
+## Logging
+
+In the integration tests, logging is implemented in a unique way. This section will go over the two distinct ways of logging:
+1. Use `logger.log` for inline logging
+2. Use `logger.pushLogs` for buffered logging
+
+Developers can use `logger.log` as a normal console log with some enhanced functionality. The main difference between `logger.log` and `console.log` is that logger log will take `LogLevel` as a parameter. Different log levels print to the console in different colors and have a minor difference in the logs which makes it easier to read the logs. There are four different types of log levels.
```

Comment:
How about the following because "major" and "minor" differences sound subjective?

> You can use `logger.log` as a default logger with enhanced functionalities.  `logger.log` supports two additional features:
> - `logger.log` takes one of the four log levels: `LogLevel.INFO`, `LogLevel.WARN`, `LogLevel.ERROR`, and `LogLevel.SUCCESS`.
> - `logger.log` outputs messages to the console in different colors based on the log level.